qcodes/dataset/dond/sweeps.py

```diff
@@ -203,3 +206,40 @@ def num_points(self) -> int:
     @property
     def post_actions(self) -> ActionsT:
         return self._post_actions
+
+
+class MultiSweep:
```

Comment:
Any input on a better name for this class is very much appreciated

Reply:
only StepTogetherSweep or TogetherSweep/SweepTogether comes to mind, feel free to use if you think it's better